ksp_plugin/interface_external.cpp

```diff
@@ -0,0 +1,199 @@
+﻿
+#include "ksp_plugin/interface.hpp"
+
+#include <string>
+
+#include "base/array.hpp"
+#include "base/status.hpp"
+#include "base/status_or.hpp"
+#include "journal/method.hpp"
+#include "journal/profiles.hpp"
+#include "physics/apsides.hpp"
+
+namespace principia {
+namespace interface {
+
+using base::Error;
+using base::StatusOr;
+using base::UniqueBytes;
+using geometry::AngularVelocity;
+using ksp_plugin::FlightPlan;
+using ksp_plugin::Navigation;
+using ksp_plugin::Vessel;
+using physics::BodyCentredNonRotatingDynamicFrame;
+using physics::ComputeApsides;
+using physics::DiscreteTrajectory;
+using physics::RigidMotion;
+using physics::RigidTransformation;
+
+namespace {
+
+// This is called "New" because it will eventually construct and release the
```

Comment:
I would call in MakeStatus for now.  Who knows what it will become once we have proper allocation/deallocation.

Reply:
Done.